ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -830,7 +834,7 @@ void postCompletionCleanup() {
 
     @Override
     GetOptions getGetOptions() {
-      return GetOptions.None;
+      return getOptions;
```

Comment:
Good point. Done. I also removed the `blobId` variable that was being passed in the constructor.